distribution/lib/Standard/Base/0.0.0-dev/src/Data/Maybe.enso

```diff
@@ -48,18 +48,4 @@ type Maybe
 
              example_is_some = Maybe.Some "yes!" . is_some
     is_some : Boolean
-    is_some = case this of
-        Nothing -> False
-        Some _ -> True
-
-    ## Check if the maybe value is `Nothing`.
-
-       > Example
-         Check if `Nothing` is `Nothing`.
-
-             import Standard.Base.Data.Maybe
-
-             example_is_nothing = Maybe.Some "yes!" . is_nothing
-    is_nothing : Boolean
-    is_nothing = this.is_some.not
-
+    is_some = this.is_nothing.not
```

Comment:
Is this 100% correct? I think likely yes but would like to make sure:
`is_nothing` is defined on `Any` type, right?
So what will `2.is_some` give? I expect, since we define `is_some` only on `Some | Nothing` it should be an unresolved symbol, but I'd like to be sure this is the case.

Reply:
```
> a = Maybe.Some 12
>>> Nothing
> a.is_some
>>> true
> Nothing.is_some
>>> false
> 3.is_some
Evaluation failed with: Method `is_some` of 3 (Integer) could not be found.
java.lang.Exception: Method `is_some` of 3 (Integer) could not be found.
        at <enso>.<eval>(Unknown Source)
        at <enso>.Debug.breakpoint(Unknown Source)
        at <enso>.Internal_Repl_Module___::Internal_Repl_Module___::internal_repl_entry_point___(Internal_Repl_Module___:4)
        at <unknown>.org.graalvm.polyglot.Value<Function>.execute(Unknown Source)

> a.is_nothing
>>> false
> Nothing.is_nothing
>>> true
```

Reply:
behaves as before and as we expect I think

Reply:
Awesome, thanks for checking this.